src/views/LogsView.js

```diff
@@ -22,6 +23,13 @@ const LogsView = ({match}) => {
     },
     pollInterval: 30000,
   });
+  const {loading: studyLoading, data: studyData} = useQuery(GET_STUDY_BY_ID, {
+    variables: {
+      kfId: match.params.kfId,
+    },
+    fetchPolicy: 'network-only',
+  });
+  const studyByKfId = studyData && studyData.studyByKfId;
```

Comment:
should this be in a separate PR ?

Reply:
This was added because we want to get the study name or study id to show on the title.